examples-gallery/intermediate/plot_parallel_park.py

```diff
@@ -199,16 +222,14 @@
 _ = prob.plot_constraint_violations(solution)
 
 # %%
-# Plot the objective function as a function of optimizer iteration.
-_ = prob.plot_objective_value()
 
 # %%
 # Show the optimal path of the mass center.
 xs, us, ps = prob.parse_free(solution)
 fig, ax = plt.subplots()
-ax.plot(xs[0], xs[1])
+ax.plot(xs[0, :], xs[1, :])
 ax.set_xlabel(r'$x$ [m]')
-ax.set_ylabel(r'$y$ [m]');
+_ = ax.set_ylabel(r'$y$ [m]');
```

Comment:
```suggestion
_ = ax.set_ylabel(r'$y$ [m]')
```

Remove the semicolon if you add the `_ =`.

Reply:
I don't know either. I thought it would suppress output in sphinx gallery.

Reply:
Anyway, it did not on my ``make html``